set.mm

```diff
@@ -7314,6 +7314,10 @@ _praeclarum theorema_ (splendid theorem).  (Contributed by NM,
   pm5.31 $p |- ( ( ch /\ ( ph -> ps ) ) -> ( ph -> ( ps /\ ch ) ) ) $=
     ( wi wa pm3.21 imim2d imp ) CABDABCEZDCBIACBFGH $.
 
+  $( Variant of ~ pm5.31 .  (Contributed by Rodolfo Medina, 15-Oct-2010.) $)
+  pm5.31r $p |- ( ( ch /\ ( ph -> ps ) ) -> ( ph -> ( ch /\ ps ) ) ) $=
```

Comment:
Shouldn't this be called `pm5.31com`? I expected `pm5.31r` to turn a biconditional into an implication, but that isn't what this is.

Reply:
It's difficult (I just took this from RM's mathbox without changing the label). "r" usually means "right", maybe because "ps" is the rightmost wff in the antecedent? Furthermore, there is no label following the pattern pm*.*com in set.mm yet. I would like to hear (at least) a third opinion about this.

Reply:
Maybe just keep the label from the mathbox. I see the analogy to `jccir`, `jccil`, `ancld`, and `ancrd` (although the left variant here isn't named with an "l"), or even more closely to `pm4.71r` (where "r" apparently stands for "reversed").
I don't exactly retract the idea that an analogy to `impcom`  would also make sense, but it doesn't seem as compelling as when I first wrote this.

Reply:
> ... or even more closely to `pm4.71r` (where "r" apparently stands for "reversed"). ...

Yes, you are right: "r" means "reversed" in this case, and this suffix is also mentioned in the conventions:

    r : theorem with something reversed (e.g., a biconditional)